apiclient/client.go

```diff
@@ -164,17 +164,26 @@ func (c *HTTPclient) Request(method string, jsonBody any, urlPath ...string) ([]
 	}
 
 	log.Debugw("http request", "type", method, "path", u.Path, "body", jsonBody)
-	resp, err := c.c.Do(&http.Request{
-		Method: method,
-		URL:    u,
-		Header: headers,
-		Body: func() io.ReadCloser {
-			if jsonBody == nil {
-				return nil
-			}
-			return io.NopCloser(bytes.NewBuffer(body))
-		}(),
-	})
+	var resp *http.Response
+	for i := 1; i <= 3; i++ {
```

Comment:
We might need to config this value (the number of iterations). 

Three is a magic number that could be not enough when performing benchmarks. 

You might make it an exported variable of the HTTPclient type and set it to 3 (from a documented constant please) on the type constructor.

Reply:
3 is definitely the most magical number ever

i opted for a c.SetRetries instead, to keep the style of the package which has no exported vars, and uses Set methods, like c.SetAuthToken